helm/hpcc/values.schema.json

```diff
@@ -246,6 +249,30 @@
       "required": [ "name", "prefix" ],
       "additionalProperties": false
     },
+    "resourceLimitItem": {
+      "type": "object",
+      "properties": {
+        "cpu": {
+          "oneOf": [
+            { "type": "string" },
+            { "type": "number" }
+          ]
+        },
+        "memory": {
+          "oneOf": [
+            { "type": "string" },
+            { "type": "integer" }
```

Comment:
what is the reason for onef here being "integer" vs "number" for cpu?

Reply:
You can request milliCPUs ...

Reply:
yes, number allows decimals and you can specify milliCPUs as decimals.  Memory can't be decimal, only integers.

Reply:
yes ok, that makes sense.